Documentation/network/l2-announcements.rst

```diff
@@ -330,3 +330,40 @@ has been reached.
 .. note::
    Since this feature has no IPv6 support yet, only ARP messages are sent, no 
    Unsolicited Neighbor Advertisements are sent.
+
+.. _l2_pod_announcements:
+
+************************************
+L2 Pod Announcements
+************************************
```

Comment:
You're formatting this as a top-level title, at the same level as the page title. It would probably look more consistent to have just one such title on the page, have you considered decreasing the level? If it should be top-level, should it be on its own page?

Reply:
Good point, I've reduced the level.